srcpkgs/volta/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'volta'
+pkgname=volta
+version=0.7.1
+revision=1
+archs="x86_64 i686"
+build_helper="rust"
+hostmakedepends="cargo pkg-config"
+makedepends="libressl-devel"
+short_desc="JavaScript tool version sync helper"
+maintainer="Alex Lohr <alex.lohr@logmein.com>"
+license="BSD-2-Clause"
+homepage="https://volta.sh/"
+distfiles="https://github.com/volta-cli/${pkgname}/archive/v${version}.tar.gz"
+checksum=e53a07e167bb64103f36901423f5a377a2ea89ecfdd7a1343e69d659f99f9c1b
+
+pre_build() {
+	cargo update --package openssl-sys --precise 0.9.53
+}
+
+do_build() {
+	cargo build --release --target ${RUST_TARGET}
+}
+
+do_install() {
+	vlicense LICENSE
+
+	vbin target/${RUST_TARGET}/release/volta
+	vbin target/${RUST_TARGET}/release/volta-shim
+	vbin target/${RUST_TARGET}/release/volta-migrate
+
+	vmkdir etc/volta 755
+	vinstall shell/unix/load.bash 644 etc/volta
+	vinstall shell/unix/load.sh 644 etc/volta
+	vinstall shell/unix/load.fish 644 etc/volta
+	vinstall ${FILESDIR}/install.sh 755 etc/volta
```

Comment:
Is `install.sh` really necessary? Looks like there is `volta setup` (`src/command/setup.rs`) which is doing that.

Reply:
Unfortunately, `volta setup` will not add the load scripts to the profiles. While you could still use volta this way, it will lose part of its functionality.

Reply:
The scripts are gone now.